test/entrypoint.sh

```diff
@@ -12,10 +12,19 @@ while ! exec 6<>/dev/tcp/0.0.0.0/3306; do
     sleep 1
 done
 
+# make sure we can reach the rabbitmq
+while ! exec 6<>/dev/tcp/0.0.0.0/5672; do
```

Comment:
Sorry, I'm not super familiar with this bash-fu. Wouldn't reusing the same fd as the mysql check maybe cause weird bugs? We do the follow up clean-up-looking execs below for a reason, maybe?